Apps/WebClient/src/NewClientApp/src/components/private/timeline/comment/CommentComponent.vue

```diff
@@ -0,0 +1,153 @@
+            isEditMode.value = false;
+        })
+        .catch(handleCommentError)
+        .finally(() => (isUpdating.value = false));
+}
+
+function removeComment(): void {
+    if (confirm("Are you sure you want to delete this comment?")) {
+        isUpdating.value = true;
+        commentStore
+            .deleteComment(userStore.user.hdid, commentComponentProps.comment)
+            .then(() => logger.info("Comment removed"))
+            .catch(handleCommentError)
+            .finally(() => (isUpdating.value = false));
+    }
+}
+</script>
+
+<template>
+    <v-sheet class="px-4 py-2 my-2" color="grey-lighten-5">
+        <v-row v-if="!isEditMode">
+            <v-col>
+                <p data-testid="commentText" class="text-body-1">
+                    {{ comment.text }}
+                    <br />
+                    <span class="text-caption text-medium-emphasis">
+                        {{ formatDate(comment.createdDateTime) }}
+                    </span>
+                </p>
+            </v-col>
+            <v-col cols="auto">
+                <v-menu data-testid="commentMenuBtn">
+                    <template #activator="{ props }">
+                        <HgIconButtonComponent
+                            icon="ellipsis-v"
+                            size="small"
+                            v-bind="props"
+                        />
+                    </template>
+                    <v-list>
+                        <v-list-item
+                            data-testid="commentMenuEditBtn"
+                            title="Edit"
+                            @click="editComment()"
+                        />
+                        <v-list-item
+                            data-testid="commentMenuDeleteBtn"
+                            title="Delete"
+                            :loading="isUpdating"
+                            @click="removeComment()"
+                        />
+                    </v-list>
+                </v-menu>
+            </v-col>
+        </v-row>
+        <v-row v-else :class="{ 'flex-column': isMobile }">
+            <v-col>
+                <v-textarea
+                    id="comment-input"
+                    v-model="commentInput"
+                    data-testid="editCommentInput"
+                    max-rows="10"
+                    rows="1"
+                    variant="underlined"
+                    auto-grow
+                    placeholder="Editing a comment"
+                    maxlength="1000"
+                    :loading="isUpdating"
+                    hide-details
+                />
+            </v-col>
+            <v-col cols="auto" class="d-flex align-center">
+                <HgButtonComponent
+                    data-testid="saveCommentBtn"
+                    class="mr-2"
+                    variant="primary"
+                    :disabled="commentInput === ''"
+                    :loading="isUpdating"
```

Comment:
```suggestion
                    :disabled="isUpdating || !commentInput"
                    :loading="isUpdating"
```

Reply:
Isn't `isUpdating` redundant on `:disabled`?

Reply:
I tested the follow in one of the Vuetify playgrounds:
```Javascript
<v-btn
        :disabled="false"
        :loading="loading"
        class="flex-grow-1"
        height="48"
        variant="tonal"
        @click="load"
      >
        Take Checkup
      </v-btn>

load() {
        this.loading = true
        console.log('CLICK')
        setTimeout(() => (this.loading = false), 3000)
      },
```

The click only registered a single time while the loader was running, even with the attached `:disabled`

Reply:
I do remember you mentioning that the :disabled overrides the :loading's disabled state, but this doesn't seem to be reflected in the experiment previously mentioned.